docs/layers/trips-layer.md

```diff
@@ -69,8 +100,8 @@ This value should be in the same units as the timestamps from `getPath`.
 Called for each data object to retreive paths.
 Returns an array of navigation points on a single path.
 Each navigation point is defined as an array of three numbers: `[longitude, latitude, timestamp]`.
-Points should be sorted by timestamp.
-
+Points should be sorted by `timestamp`. 
+Because `timestamp` is represented as 32-bits floating number, raw unix epoch can not be used.
```

Comment:
Add: "You may test the validity of a timestamp by calling `Math.fround(t)` to check if there would be any loss of precision."